continuous_integration/appveyor/requirements.txt

```diff
@@ -1,18 +1,9 @@
-# Fetch numpy and scipy wheels from the sklearn rackspace wheelhouse.
-# Those wheels were collected from http://www.lfd.uci.edu/~gohlke/pythonlibs/
-# This is a temporary solution. As soon as numpy and scipy provide official
-# wheel for windows we ca delete this --find-links line.
---find-links http://28daf2247a33ed269873-7b1aad3fab3cc330e1fd9d109892382a.r6.cf2.rackcdn.com/
-
-# fix the versions of numpy to force the use of numpy and scipy to use the whl
-# of the rackspace folder instead of trying to install from more recent
-# source tarball published on PyPI
-numpy==1.11.3
-scipy==0.18.1
+#
+# /continuous_integration/install_wheels.py takes care of installing numpy and scipy.
```

Comment:
are you sure than this still required? because you get rid py27 (if I remember correctly), I guess it should work without "hacks"

Reply:
You're right. I've removed this install_wheels bollocks altogether. If it works, we can "just" build with the latest versions of scipy and numpy, and expect them to have wheels available. Let's see if it builds.